### PR TITLE
Update navbar

### LINKED ISSUE #39: Implement updated Figma Navbar styling.  Confirm consistency of design plans across each view.
This is a sub-issue of Issue #14 

- [ ] Navbar should be visible at the bottom center of all views

UI Wireframe from FIgma:
Screen Shot 2021-05-22 at 7 49 19 PM

Screen Shot 2021-05-22 at 10 46 34 PM

### PR DESCRIPTION
**Pull request by jamesncox**
## Description

Implement [Figma Navbar design](https://www.figma.com/file/Gd4a2z8c5iNG6b3ZWs36g4/Smart-Shopping-List?node-id=0%3A1) in `List.js` and `AddItem.js`.

## Related Issue

Closes #39 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|   ✓| :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

1. Move `<NavBar />` component in `App.js` to bottom of `</Switch>`.
2. Apply Tailwind utility classes to make the nav bar stay at the bottom of the screen and mimic the breakpoints applied to the `List.js` and `AddItem.js` components.`
```javascript
<nav className="grid grid-cols-2 bg-blue-ncs p-5 text-lg md:text-xl w-full divide-x-2 divide-gray-200 absolute bottom-0 md:px-36 lg:px-48 xl:px-72 lg:w-2/3">
```
3. Use two of our [Heroicons](https://heroicons.com/) SVGs as JSX inside the two `RouterLink`s. 
4. Wrap the Heroicons in `<span>` to apply accessibility for screen readers.
e.g.
```javascript
 <span
  role="img"
  aria-label="Your List"
  className="flex justify-center items-center"
 >
```
5. Apply hover effect on navigation links and persists highlighted color on `activeClassName`
```javascript
 <RouterLink
  className="hover:text-orange-yellow"
  exact
  activeClassName="text-orange-yellow"
  to="/add-item"
>
```

### Visual Changes

| | Before | After |
| --- | --- | --- |
| Mobile Nav | before-nav-mobile | after-nav-mobile |
| Desktop | before-nav-desktop | after-nav-desktop |

## Testing Steps / QA Criteria

1. On `main` run `git pull`
2. Run `git checkout update-navbar`
3. Run `npm install`
4. Run `npm start`
5. Switch back and forth between `AddItem` and `List` and notice how the highlighted yellow color remains on the selected page.
6. Tab through the page to highlight the different links and navigate back and forth between the pages.

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -2,23 +2,62 @@ import { NavLink as RouterLink } from 'react-router-dom';
 
 export default function NavBar(props) {
   return (
-    <nav>
+    <nav className="grid grid-cols-2 bg-blue-ncs p-5 text-lg md:text-xl w-full divide-x-2 divide-gray-200 absolute bottom-0 md:px-36 lg:px-48 xl:px-72 lg:w-2/3">
       <RouterLink
-        className="nav-link"
+        className="hover:text-orange-yellow"
```
Comment (jssckbl) [inline src/components/NavBar.js:7]: Yes hover color!

File: src/components/NavBar.js [hunk 2]
```diff
@@ -2,23 +2,62 @@ import { NavLink as RouterLink } from 'react-router-dom';
 
 export default function NavBar(props) {
   return (
-    <nav>
+    <nav className="grid grid-cols-2 bg-blue-ncs p-5 text-lg md:text-xl w-full divide-x-2 divide-gray-200 absolute bottom-0 md:px-36 lg:px-48 xl:px-72 lg:w-2/3">
```
Comment (jssckbl) [inline src/components/NavBar.js:5]: The nav is a different shade of blue than the Figma, but I think it sticks out more the way you have it here without being too much. All that to say, I like it.

**Comment by jssckbl** [comment]
Looks ready to merge to me!

File: src/components/NavBar.js (hunk 2 above)
Comment (jamesncox) [reply src/components/NavBar.js:5]: That's interesting! The color code from [our palette](https://coolors.co/ef476f-ffd166-06d6a0-118ab2-073b4c) is `#118AB2` which matches the hex code in our `tailwind.config.js` file:

```javascript
 colors: {
  'paradise-pink': '#EF476F',
  'orange-yellow': '#FFD166',
  'caribbean-green': '#06D6A0',
  'blue-ncs': '#118AB2', // same as the color palette hex code
  'midnight-green': '#073B4C',
},
```

But Figma does appear to represent it a bit differently. Weird!

Comment (jssckbl) [reply src/components/NavBar.js:5]: I checked and it looks like in Figma it is the same color being used, but a transparency was also used in layering with the background color.

**Pull request merged**